### DISCUSSION TITLE
PBR materials to SDF

### DISCUSSION THREAD

**Discussion by brothermechanic** [dfki-ric/phobos#280] [Q&A]
Is it possible model's PBR material with texture pack like Diffuse+Normal+OcclusionRoughnessMetall 
transfer to SDF description by Phobos?

**Comment by hwiedPro** [comment]
Hi @brothermechanic,
basically yes (as far as SDF supports all this). But it will need some implementation work:
- _phobos/blender/blender2phobos.py_: deriveMaterial() has to be adapted to understand the PBR material definition
- _phobos/io/representation.py_: class Material has to be reviewed whether all those information can there be stored.
- _phobos/data/xmlformats.json_: The SDF definition has to be touched to define how all those properties have to be written out from the Material class to SDF
- _phobos/blender/phobos2blender.py_: createMaterial() has to be adapted so that importing those data from SDF will work as well.
